integration_tests/keysend_test.go

```diff
@@ -78,8 +78,10 @@ func (suite *KeySendTestSuite) TearDownSuite() {
 }
 
 func (suite *KeySendTestSuite) TestKeysendPayment() {
+	suite.service.Config.ServiceFee = 1
```

Comment:
yes, this is not nice. but I don't know how to best do it. 
We don't want to enabled it for all because some tests assume it's not enabled (e.g. expect certain tx entries)